salesforce/10.9/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -43,6 +43,16 @@ These two structures ensure that no message is processed twice and that when the
 
 For each message that comes through a topic or streaming channel to which the connector is subscribed, the connector updates the object store information using up to four transactions.
 
+[NOTE]
+---
```

Comment:
(Oops...a CU edit...):
This is probably my confusion in seeing this out of context, but I am wondering if topic/streaming channel (above), multiple workers, and multiple instances all refer to the same thing: is a worker also an instance or an app?

I'm looking for a simpler wording around these points:

* You have multiple xyz
* Each xyz will process each message, creating overhead
* Avoid redundant processing by...
** filter duplicates
** have one app "deliver the mail"?